mmv1/products/lustre/Instance.yaml

```diff
@@ -0,0 +1,133 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Instance
+description: Description
+base_url: projects/{{project}}/locations/{{location}}/instances
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+create_url: projects/{{project}}/locations/{{location}}/instances?instanceId={{instance_id}}
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+examples:
+  - name: lustre_instance_basic
+    primary_resource_id: 'instance'
+    vars:
+      name: 'my-instance'
+      address_name: 'my-ip-address-name'
+      network_name: 'my-network'
+      subnetwork_name: 'my-subnetwork'
+    test_vars_overrides:
+      'network_name': 'acctest.BootstrapSharedServiceNetworkingConnection(t, "default-vpc")'
```

Comment:
The preferred naming is `PRODUCT-RESOURCE-network-config`, but it should definitely be unique per test (unless you have issues with doing that)

Reply:
Spoke offline we will rename this in follow up CL as this requires quota changes.